pkg/controllers/provisioning/suite_test.go

```diff
@@ -923,22 +932,153 @@ var _ = Describe("Provisioning", func() {
 							Values:   []string{"node-name"},
 						},
 					},
-					ResourceRequirements: corev1.ResourceRequirements{Requests: corev1.ResourceList{corev1.ResourceCPU: resource.MustParse("4"), corev1.ResourceMemory: resource.MustParse("4Gi")}},
+					ResourceRequirements: corev1.ResourceRequirements{Requests: corev1.ResourceList{corev1.ResourceCPU: resource.MustParse("1"), corev1.ResourceMemory: resource.MustParse("1Gi")}},
```

Comment:
Why do the resource requirements not meet the DS here? Is this intentional?

Reply:
Yes, it's intentional -- comment at the top of the test states that we should be using the pod representation for resource requests but need to use the DaemonSet representation for affinity

Reply:
Got it, makes sense. I am wondering though, is this the correct behavior? What scenarios are there where the resource requests on the pod will differ from the owning daemonset. I imagine it would be after an update to the daemonset, but shouldn't we respect the updated value rather than the value on current pods since that will be what's provisioned for the new Node?

Reply:
> resource requests on the pod will differ from the owning daemonset

It's in the comment, LimitRange is one example